app/controllers/recipes_controller.rb

```diff
@@ -23,6 +23,7 @@ def index
   end
 
   def show
+    @has_rated = @recipe.ratings.any? {|rating| rating.user == current_user}
```

Comment:
Why are you doing this? I added a `#rated_by?` instance method you can use in the view (please red through the recipe_spec and model)

Reply:
Shall we use ```#rated_by?``` instead of ```@has_rated``` ?